tests/e2e/logging_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2024 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"go.etcd.io/etcd/tests/v3/framework/e2e"
+)
+
+func TestNoErrorLogsDuringNormalOperations(t *testing.T) {
+	tests := []struct {
+		name          string
+		clusterSize   int
+		allowedErrors map[string]bool
+	}{
+		{
+			name:          "single node cluster",
+			clusterSize:   1,
+			allowedErrors: map[string]bool{},
```

Comment:
I do not get time to dig out why 3.5 doesn't see the error message as 3.6. Did you ever dig into it?

Reply:
3.5: https://github.com/ghouscht/etcd/blob/release-3.5/server/embed/etcd.go#L882-L892

main: https://github.com/ghouscht/etcd/blob/main/server/embed/etcd.go#L872-L885

Added with https://github.com/etcd-io/etcd/pull/14116

Reply:
Thanks.